pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -465,6 +465,9 @@ type ClusterNetworkingConfig struct {
 	// Defaults to ipvs.
 	ProxyMode string `json:"proxyMode"`
 
+	// IPVS defines kube-proxy ipvs configuration options
+	IPVS IPVSConfiguration `json:"ipvs"`
```

Comment:
One thing I would consider here is adding `omitempty` param. If the proxy mode is different than `ipvs` then it would be nice to not return this at all. WDYT?

Reply:
omitempty doesn't work on structs, only on pointers to structs